doc/source/reference/groupby.rst

```diff
@@ -72,6 +72,7 @@ Computations / descriptive stats
    GroupBy.prod
    GroupBy.rank
    GroupBy.pct_change
+   GroupBy.rows
```

Comment:
let's not add just yet (e.g. keep this non-exposed until api decisions)

Reply:
Done